### ISSUE TITLE
add retries to watcher for JAWS functions

### ISSUE THREAD

**Issue by aclum** [microbiomedata/nmdc_automation#481]
We had an issue with the JAWS server which ended the watcher on 4/18/25

2025-04-18 11:35:38,076 ERROR: Failed to Submit Job: Unable to communicate with JAWS server: ('Connection aborted.', RemoteDisconnected('Remote end closed connection
without response'))

We should add retries with increasingly long intervals similar to what we have for posting to runtime.

**Comment by aclum** [comment]
Separately the JAWS team increased the number of workers and the RMQ connection
